Gemfile

```
@@ -39,6 +39,7 @@ gem 'versionist'
 gem 'redis-rails'
 gem 'sidekiq', '~> 5.2.7'
 gem 'pundit', '~> 2.0.1'
+gem 'httparty'
```

We could lock the version
```suggestion
gem 'httparty', '~> 0.16.4'
```